demo/src/test/java/com/example/test/pages/KBBMOOCProject/ButtonStyles.java

```diff
@@ -0,0 +1,20 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ButtonStyles extends KbbBasePage {
+
+    public ButtonStyles(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+     WebElement btnStyles = keyword.findWebElementByXpath("//div[@label='Styles']");
```

Comment:
should write the web element locator in separate line then call into a method/function